app/bundles/CoreBundle/Controller/ExceptionController.php

```diff
@@ -77,8 +77,7 @@ public function showAction(Request $request, \Throwable $exception, DebugLoggerI
         }
 
         $template   = "MauticCoreBundle:{$layout}:{$code}.html.twig";
```

Comment:
@escopecz I am not done with the controller yet, but I want to make this small change to this controller, it will be touched again after I get rid of templating engine. I just made some changes to remove the templating helper

Reply:
@escopecz this is done now in this #12124, please merge it first then merge this PR